tsdb/index/index.go

```diff
@@ -722,9 +725,7 @@ func (w *Writer) writeLabelIndexesOffsetTable() error {
 		return err
 	}
 
-	w.buf1.Reset()
-	w.buf1.PutHashSum(w.crc32)
-	return w.write(w.buf1.Get())
+	return w.writeLenghtAndHash(startPos)
```

Comment:
This looks wrong to me. Isn't the length written twice?

Reply:
Should be fixed by https://github.com/prometheus/prometheus/pull/13509.

Reply:
Exactly, I did this wrong and has to be fixed

Reply:
Oh damn! I missed this! Thanks folks for spotting it.